src/navigation/guideNavigation.js

```diff
@@ -11,17 +10,12 @@ const nav = [
   { title: 'Developers' },
 ];
 
-const collections = await Promise.all([
+const content = await Promise.all([
```

Comment:
@alexBinns Change: Add a test plan and deploy to the dev environment so we can check the changes 👍